metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -217,9 +238,9 @@ public void apply(final Metafix metafix, final Record record, final List<String>
         public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
             final String joinChar = options.get("join_char");
             final Value newValue = new Value(params.subList(1, params.size()).stream()
-                    .filter(f -> literalString(f) || record.get(f) != null)
```

Comment:
And: Why the changed indentation everywhere?

Reply:
yep, right. Sorry - struggling with IDE settings and got a bit inattentive :( .. changed that